### DISCUSSION TITLE
Obvious DDI in AUC

### DISCUSSION THREAD

**Discussion by HaiXxxxxx** [Open-Systems-Pharmacology/Forum#1753] [Questions & Problems]
If a potentiator can increase the AUC of a drug by over a hundredfold, what special characteristics might affect it?

**Comment by prvmalik** [comment]
Victim compound would have to be low bioavailability due to significant gut / first pass metabolism (<4%), which is then almost completely diminished.

**Reply by HaiXxxxxx** [reply]
Thank you!
